CONTRIBUTING.md

```diff
@@ -137,23 +137,22 @@ Committers!
 With great power comes great responsibility yada yada yada:
 
  * Code is peer reviewed, you should (almost) never push your own code.
```

Comment:
should this not be "merged" instead of pushed now?

Reply:
although i don't 100% agree - I've updated to say:

>  * Code is peer reviewed, you should not merge un-reviewed code.

This is probably enforced by github already, so I'd also be happy to just delete this.